packages/flutter/lib/src/material/page_transitions_theme.dart

```diff
@@ -325,7 +325,11 @@ class _ZoomEnterTransition extends StatelessWidget {
       },
       child: FadeTransition(
         opacity: fadeTransition,
-        child: ScaleTransition(scale: scaleTransition, child: child),
+        child: ScaleTransition(
+          scale: scaleTransition,
+          filterQuality: FilterQuality.none,
```

Comment:
without knowing the background this seems strange. Should we add comments? Or is the idea that the docs on ScaleTransition cover this?

Reply:
What seems strange?

Reply:
Its documented on those widgets that it uses an ImageFilter if one is provided, though the documentation isn't extensive: https://api.flutter.dev/flutter/widgets/ScaleTransition/filterQuality.html

Reply:
I would have chosen FilterQuality.low. I don't think there will be much performance difference, but low will be smoother.